### DISCUSSION TITLE
`em` and `ex` as length units

### DISCUSSION THREAD

**Discussion by pr-apes** [speedata/publisher#379] [Q&A] [answered]
@pgundlach,

I have the following definition:

```` xml
<DefineTextformat name="text" indentation="15mm" padding-top="12pt" />
````

I wonder whether this cannot be related to the size of the font used.

If the main font (size) has to be changed, all lengths given in absolute units have to be checked again.

Is there no way to use `em` and `ex` as length units?

Many thanks for your help.

**Comment by pgundlach** [comment, marked as answer]
Currently this is not possible. Sizes get fixed the moment the definition is processed.

**Reply by pr-apes** [reply]
It is really a pity. But it would be a great feature to have in the (not–so–distant 😅) future.

Many thanks for your help.
